### PR TITLE
provider/marathon: ensure the app_id is valid path

### PR DESCRIPTION
**Pull request by rjeczalik**

### CODE REVIEW

File: go/src/koding/kites/kloud/provider/marathon/stack.go [hunk 1]
```diff
@@ -159,6 +159,11 @@ func (s *Stack) convertInstancesToGroup(name string, app map[string]interface{})
 	appID, ok := app["app_id"].(string)
 	if !ok || appID == "" {
 		appID = name
+
+		if appID[0] != '/' {
```
Comment (ppknap) [inline go/src/koding/kites/kloud/provider/marathon/stack.go]: could you: `appID = path.Join("/", appID)` ?

**Review by ppknap — APPROVED** [review]

File: go/src/koding/kites/kloud/provider/marathon/stack.go (hunk 1 above)
Comment (rjeczalik) [inline go/src/koding/kites/kloud/provider/marathon/stack.go]: Done.

**Review by ppknap — APPROVED** [review]
LGTM

**Review by gokmen — APPROVED** [review]

**Pull request merged**